### DISCUSSION TITLE
Need help in testing the application on different gpu's - Vulkaninfo

### DISCUSSION THREAD

**Discussion by arunsivaramanneo** [arunsivaramanneo/GPU-Viewer#31] [General]
Hi All

I am trying my best to get the application tested on all 3 vendors currently i own a MSI laptop which has Nvidia optimus setup with Intel HD Graphic 630 (kabylake) and an Nvidia GTX 1050 Ti GPU and a AMD Matebook 13 which has the AMD Ryzen 5 with APU. My testing is limited to these GPU's , though i cover all 3 major vendors these hardware is almost outdated and may not get new features thats been on the Frequently updated Vulkan tools and Graphic drivers. Hence i request whoever is using the application to send me a report of vulkaninfo for me to do better testing .

**Comment by unexploredtest** [comment]
Sure
my PC, which has AMD Radeon HD 7770:
[vulkaninfo.txt](https://github.com/arunsivaramanneo/GPU-Viewer/files/6386267/vulkaninfo.txt)
I also have a laptop containing an AMD APU and an AMD discrete card, will post soon
